README.md

```diff
@@ -30,8 +30,6 @@ Please do note that:
 - This module was written from a browser environment so it depends on window and
   document to be available in the global scope. We do fallback to a polyfill if
   those are missing so the module should not break in a Node.js environment.
-- The module is written in ES6, so make sure that you environment supports this
```

Comment:
this is still true, only part that needs to be removed it the transpiler section